Dockerfile

```diff
@@ -5,7 +5,14 @@ WORKDIR /usr/src/app
 RUN yarn config set network-timeout 600000 -g
 RUN yarn install && yarn build
 
-FROM --platform=linux/amd64 docker.io/library/nginx:stable
+FROM --platform=linux/amd64 registry.redhat.io/rhel8/httpd-24
```

Comment:
Do you want to specify the version ?

Something like `registry.redhat.io/rhel8/httpd-24:1-226`
or 
`registry.redhat.io/rhel8/httpd-24:1-230` ?

It depends on how confident we are that the new version of httpd does not cause any breaking changes for us.

Reply:
This is copied from kam, not sure if new version would cause any breaking changes for us.

Reply:
Okay . Let's use it in that case.